src/style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  position: sticky;
+  height: 60px;
+  top: 0;
+  display: flex;
+  /* padding: 10px 50px 10px 22.5px; */
+  justify-content: space-between;
+  align-items: center;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  box-shadow: 5px 0 3px 1px lightgray;
+  color: black;
+}
+
+.image__container {
+  width: 40;
+  height: 40;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  margin-right: 50px;
```

Comment:
You could use `padding: 0 50px` for your header instead of giving margins for logo and nav